src/javaExercises/stairCase/StairCase.java

```diff
@@ -0,0 +1,22 @@
+package javaExercises.stairCase;
+
+import java.util.Scanner;
+
+public class StairCase {
+    public static void main(String[] args) {
+
+        Scanner sc = new Scanner(System.in);
+        System.out.println("Input desired staircase height:");
+        int n = sc.nextInt();
+
+        for (int i = 1; i <= n; i++) {
+            for (int j = 1; j <= n; j++) {
+                if ((i + j) > n) {
+                    System.out.print("#");
```

Comment:
extract all this code into a separate method that will return a ***StringBuffer*** and call that method from main method,
smth like this: `System.out.println(returnStairs(numberOfStairs));`

Reply:
Done